### ISSUE TITLE
Media won't play unless uMatrix is disabled entirely.

### ISSUE THREAD

**Issue by uBlock-user** [uBlockOrigin/uMatrix-issues#164]
### Prerequisites

- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
    - [x] I also searched the existing issues at <https://github.com/gorhill/uMatrix/issues>
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uMatrix](https://old.reddit.com/r/uMatrix/)
- I tried to reproduce the issue when...
    - [x] uMatrix is the only extension
    - [ ] uMatrix with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of uMatrix
- [x] I checked the [documentation](https://github.com/gorhill/uMatrix/wiki) to understand that the issue I report is not a normal behavior
- [x] I used the logger to rule out that the issue is caused by my ruleset

### Description

Same as the title.

### A specific URL where the issue occurs

`https://openload.co/embed/arbt8YpMPdM/?autostart=true`

### Steps to Reproduce

1 ) Open the dev tools on Chromium, click on the Network tab and set the network request to **Media** instead of **All**, now browse to `https://openload.co/embed/arbt8YpMPdM/?autostart=true` and click the play button, the spinner will keep spinning, notice the a media request appears in the network tab and the status remains **Pending** and consequently video won't load and play.

2 ) Disable _Spoof Referrer Header_ switch and refresh the page, click the play button and now notice the Network tab where a media request appears as **Pending** again and consequently video won't load and play.

3 ) Disable uMatrix entirely on `openload.co` with `matrix-off: openload.co true`, disable all the four switches and refresh the page, click the play button and notice the dev tools network tab for a media request which will still appear as **Pending** for the third time and consequently video won't load and play.

4 ) Disable uMatrix itself through `chrome://extensions` page, refresh the page, click the play button, notice the Network tab of dev tools which the media request appears, but this time it's fetching the media and the status is **206** instead of **Pending** and consequently video loads and plays normally as expected finally.

So basically I cannot play the media unless I disable uMatrix entirely on Chromium on openload. 

### Ruleset

```
* openload.co frame allow
* openload.co media allow
* openload.co script allow
* openload.co xhr allow
* oloadcdn.net media allow
* oloadcdn.net xhr allow
```

### Supporting evidence

Media request playing normally without uMatrix with 206 status -- 

![](https://i.gyazo.com/80df25b3161a28263dc92b1c2c670bfc.png)

Media request stuck at **Pending** with uMatrix enabled/whitelisted on site -- 

![](https://i.gyazo.com/3c2a7269b62162cb0bc240f2f33a53c8.png)

### Your environment

* uMatrix version: 1.3.17.104
* Browser Name and version: Chromium 77
* Operating System and version: Win 10

**Comment by gorhill** [comment]
I added `openload.co oloadcdn.net media allow` to your ruleset and all worked fine.

**Comment by gwarser** [comment]
Works on 75

**Comment by uBlock-user** [comment]
Updated the ruleset. It won't play for me whatsoever.

**Comment by uBlock-user** [comment]
Stays stuck like this --

![](https://i.gyazo.com/2548bda2548d85d987b94d6c9b32bac8.png)

**Comment by gorhill** [comment]
Ok I can reproduce with 1.3.17.104, I realized I still had 1.3.17.102 in my Chrome 77 version.

**Comment by uBlock-user** [comment]
On my end, I can reproduce going as far as 1.3.17.101 on `openload.co/pw`

**Comment by uBlock-user** [comment]
Can't no longer reproduce on Chromium 79 with 1.3.17.104.

Consider this fixed.
